### DISCUSSION TITLE
Problem with new web browsers versions (chrome, firefox)

### DISCUSSION THREAD

**Discussion by FantFRS** [tchiotludo/akhq#1390] [Q&A] [answered]
Hi everybody,

Anyone have an idea how to by pass https redirect in new web browsers versions.
I'm using Chrome version 110 and the http url which usually works before updating browser version is switched automatically to https and throws ERR_SSL_PROTOCOL_ERROR.

Need your help please.

Thank you.

**Comment by FantFRS** [comment]
Is it possible to implement HTTPS to acess akhq ui?

Thank you.

**Comment by tchiotludo** [comment, marked as answer]
You can enable https with [micronaut configuration](https://docs.micronaut.io/latest/guide/index.html#dualProtocol)
